docs/en/admins/08_FeedUpdates.md

```diff
@@ -1,19 +1,56 @@
 # Setting Up Automatic Feed Updating
 
-FreshRSS updating is controlled by a script located at `./app/actualize_script.php`. Knowing this, we can create a Cron job to launch the update script.
+FreshRSS updating is controlled by a script located at `./app/actualize_script.php`. Knowing this, we can trigger the script periodically to have up-to-date feeds.
 
-**Note:** the update script will not fetch feeds more often than every twenty minutes, so there's no sense in setting the Cron job to run anymore often than that.
+**Note:** the update script will not fetch feeds more often than every twenty minutes, thus triggering on a shorter period is counterproductive.
 
-You will need to check the Cron documentation for your specific distribution ([Debian/Ubuntu](https://help.ubuntu.com/community/CronHowto), [Red Hat/Fedora/CentOS](https://fedoraproject.org/wiki/Administration_Guide_Draft/Cron), [Slackware](https://docs.slackware.com/fr:slackbook:process_control?#cron), [Gentoo](https://wiki.gentoo.org/wiki/Cron), [Arch Linux](https://wiki.archlinux.org/index.php/Cron)...) to insure you set the Cron job correctly.
+**Note:** in the following examples, we assume that FreshRSS is installed in `/usr/share/FreshRSS`. You'll need to modify the path to FreshRSS to reflect your own system.
+
+## Cron as a trigger
+
+You will need to check the Cron documentation for your specific distribution ([Debian/Ubuntu](https://help.ubuntu.com/community/CronHowto), [Red Hat/Fedora/CentOS](https://fedoraproject.org/wiki/Administration_Guide_Draft/Cron), [Slackware](https://docs.slackware.com/fr:slackbook:process_control?#cron), [Gentoo](https://wiki.gentoo.org/wiki/Cron), [Arch Linux](https://wiki.archlinux.org/index.php/Cron)...) to make sure you set the Cron job correctly.
 
 It's advisable that you run the Cron job as your Web server user (often `www-data`).
 
-## Example on Debian/Ubuntu
+### Example on Debian/Ubuntu
 To run the updater script every hour, and 10 minutes past the hour:
 
 Run `sudo crontab -e` and copy the following line into the crontab:
 ```
 10 * * * * www-data php -f /usr/share/FreshRSS/app/actualize_script.php > /tmp/FreshRSS.log 2>&1
 ```
 
-This crontab example, of course, assumes that FreshRSS is installed in `/usr/share/FreshRSS`; if you've installed it somewhere else, be sure to correct the path in your crontab entry.
+## Systemd as a trigger
+
+Some system can't use a cron job, but can use systemd. It's easy to configure it to mimic cron features.
```

Comment:
Inconsistent capitalization. `Cron` (as in "check the Cron documentation") or `cron`?
```suggestion
Some system can't use a Cron job, but they can use systemd. It's easy to configure it to mimic Cron's features.
```